bin/setup

```diff
@@ -48,19 +48,23 @@ FileUtils.chdir APP_ROOT do
 
   unless File.exist?('.env')
     log "== Setup .env file from .env.example =="
-    cp '.env.example', '.env'
+    system("cp .env.example .env")
   end
 
   log "== Preparing database =="
   log "⚠️  If you use docker to run postgres, make sure your database is running ⚠️"
-  system! 'bin/rails db:setup'
+  system! 'bin/rails db:drop db:create db:migrate'
+  system! 'bin/rails db:seed'
 
   log "== Removing old logs, tempfiles, and jobs =="
   system! 'bin/rails log:clear tmp:clear jobs:clear'
 
   log "== Restarting application server =="
   system! 'bin/rails restart'
 
+  log "== Precompiling assets =="
+  system! 'bin/rails assets:precompile'
```

Comment:
Motivation behind this change: https://www.loom.com/share/d5c18c74b570400fb213210c55e5c196

Reply:
Thanks! I'am very curious though why webpacker needs to do this assets:precompile step. I would think normally you don't need to do this in non-prod settings.

Ultimately, I think adding this to bin/setup is faster than figuring out what is wrong with our build/assets